### ISSUE TITLE
EPIC: Rework Apps and Marketplace

### ISSUE THREAD

**Issue by kwwii** [rancher/dashboard#6402]
Design and implement UI improvements to the Apps and Marketplace based on an evaluation of our competitors and previous functionality.

**Comment by gaktive** [comment]
Uncertain how relevant this epic still is in light of recent design work but moving to a more relevant milestone to then triage.

**Comment by kwwii** [comment]
Agreed. this is being tracked in the UX repo apps epic:"Apps and Marketplace Redesign" and the resulting development cards in the dashboard repo.

**Comment by nwmac** [comment]
Marking this epic as done - there is another epic for this that can track work not done in 2.12 that was pushed to 2.13
